api/types/client.go

```diff
@@ -57,6 +57,14 @@ type ContainerExecInspect struct {
 	Pid         int
 }
 
+// ContainerExecStartOptions holds the options to start container's exec
+type ContainerExecStartOptions struct {
```

Comment:
I realize we already have a bunch of these in the top `api/types` package, but I've been trying to move more types into sub packages that are specific to what it's related to, to allow some code to only import things related to that, e.g. `container.StopOptions`; https://github.com/moby/moby/blob/74286cba8c2c14d5360caec44d2da74858e9c707/api/types/container/config.go#L16-L17

Given that this is a new type, perhaps we should move this into `api/types/container` (and name it `container.ExecStartOptions`)